MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -64,6 +65,38 @@ def store_issue_details(issue, repo, db):
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	issue_quoted_jenkins_links = content_extracted[0]
+	issue_quoted_test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+	if issue_quoted_test_names != []:
+    		issue_details['issue_quoted_test_names'] = issue_quoted_test_names
+
+	if (jenkins_links and testnames):
```

Comment:
where did we define jenkins_links and testnames?

Reply:
Sorry @llxia , I was transferring the changes from my local test folder, that's why I missed it. It was a silly mistake from my side.